tests/modules/programs/less/correct-option-order.nix

```diff
@@ -0,0 +1,20 @@
+{
+  programs.less = {
+    enable = true;
+    options = [
```

Comment:
I'd recommend trying to use both a list and attrset with `mkBefore`/`mkAfter`.

But otherwise LGTM

Reply:
Oh, I see. Ok, I restored the coercion to a list and added `mkAfter` to the test.